test/built-ins/Temporal/Calendar/prototype/dateAdd/argument-string-invalid.js

```diff
@@ -0,0 +1,49 @@
+// Copyright (C) 2022 Igalia S.L. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+esid: sec-temporal.calendar.prototype.dateadd
+description: RangeError thrown if an invalid ISO string is used as a PlainDate
+features: [Temporal, arrow-function]
+---*/
+
+const invalidStrings = [
+  "",
+  "invalid iso8601",
+  "2020-01-00",
+  "2020-01-32",
+  "2020-00-01",
+  "2020-13-01",
+  "-999999-01-01",
+  "+999999-01-01",
```

Comment:
Are those invalid iso strings or just failing ISODateTimeWithinLimits? If the latter, please clarify the description above

Reply:
I guess these could go in a separate test.

Reply:
so you'd like the description to say something like "RangeError thrown if an invalid ISO string (or valid ISO string outside the supported range) is used as a PlainDate"?

Reply:
That would work for me, yeah.